### ISSUE TITLE
Combined (multi-band) files need to be consistent with original k-distribution files

### ISSUE THREAD

**Issue by RobertPincus** [pernak18/g-point-reduction#8]
@pernak18 We're starting to deploy the reduced-resolution k-distribution files in user scenarios. One stumbling block is that the formats of the file is not quite the same - variables don't have the same types (`int` vs `int64` for many variables; `double` -> `int64` for `temperature_Planck` in the LW files). Three scalar variables are also missing from the SW file (`tsi_default`, `mg_default`, `sb_default`). 

xarray allows you to specify encodings and we should ensure that the files are essentially identical in format.

**Comment by pernak18** [comment]
with assistance from https://github.com/earth-system-radiation/rte-rrtmgp/issues/103, i believe this is issue is addressed with the [`encode_reduction.py`](https://github.com/pernak18/g-point-reduction/blob/sw-dev/encode_reduction.py) script. this is a workaround for now, because the procedure is to run the _g_-point reduction algorithm first, then use its output in the encoding script. eventually, i will incorporate this into the [`kDistOpt()`](https://github.com/pernak18/g-point-reduction/blob/sw-dev/by_band_lib.py#L1242) method once i clean up that mess so `encode_reduction.py` is not required to be run subsequently.

**Comment by RobertPincus** [comment]
@pernak18 Any progress on this?

**Comment by pernak18** [comment]
@RobertPincus no. i didn't think more needed to be done immediately. we have a working script

**Comment by RobertPincus** [comment]
@pernak18 Can you verify that the multi-band k-distribution  files produced by the script include the three scalars for the SW and have the same types, as in the original issue?

**Comment by pernak18** [comment]
@RobertPincus sorry -- looks like i forgot about the SW

**Comment by RobertPincus** [comment]
@rick SW file you provided over Slack works. Eventually it would be great to have the "description" metadata propagate, but you could make that a separate issue once the released code produces files that RRTMGP can run with
